configs/terraform/environments/prod/gcp-workfload-identity-federation.tf

```diff
@@ -20,21 +20,26 @@ module "gh_com_kyma_project_workload_identity_federation" {
   sa_mapping = {
     "terraform_executor_pull_prod_plan" = {
       sa_name   = "projects/${data.google_client_config.gcp.project}/serviceAccounts/${google_service_account.terraform_executor.email}"
-      attribute = "subject/repository_id:147495537:repository_owner_id:39153523:workflow:pull-plan-prod-terraform"
+      attribute = "subject/repository_id:147495537:repository_owner_id:39153523:workflow:Pull Plan Prod Terraform"
+    },
+    "terraform_executor_post_prod_apply" = {
+      sa_name   = "projects/${data.google_client_config.gcp.project}/serviceAccounts/${google_service_account.terraform_executor.email}"
+      attribute = "subject/repository_id:147495537:repository_owner_id:39153523:workflow:Post Apply Prod Terraform"
     }
   }
 }
 
-resource "github_actions_variable" "gcp_terraform_executor_service_account_email" {
-  provider      = github.kyma_project
-  repository    = "test-infra"
-  variable_name = "gcp_terraform_executor_service_account_email"
-  value         = google_service_account.terraform_executor.email
-}
+# TODO(kacpermalachowski): Create custom PAT: https://stackoverflow.com/questions/77214899/permission-needed-to-create-an-environment-from-a-github-actions-workflow
+# resource "github_actions_variable" "gcp_terraform_executor_service_account_email" {
+#   provider      = github.kyma_project
+#   repository    = "test-infra"
+#   variable_name = "gcp_terraform_executor_service_account_email"
+#   value         = google_service_account.terraform_executor.email
+# }
 
-resource "github_actions_variable" "gh_com_kyma_project_gcp_workload_identity_federation_provider" {
-  provider      = github.kyma_incubator
-  repository    = "test-infra"
-  variable_name = "gh_com_kyma_project_gcp_workload_identity_federation_provider"
-  value         = module.gh_com_kyma_project_workload_identity_federation.provider_name
-}
+# resource "github_actions_variable" "gh_com_kyma_project_gcp_workload_identity_federation_provider" {
+#   provider      = github.kyma_project
+#   repository    = "test-infra"
+#   variable_name = "gh_com_kyma_project_gcp_workload_identity_federation_provider"
+#   value         = module.gh_com_kyma_project_workload_identity_federation.provider_name
+# }
```

Comment:
Do you need to keep this commented code? It can be takes from git history if it will be needed.